salesforce/10.10/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -72,367 +74,242 @@ Subscribe to a topic. This source provides topic-related notification settings f
 
 [[add-connector-operation]]
 == Add a Connector Operation to the Flow
 
 When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
 
-To add an operation for Salesforce, follow these steps:
+To add an operation for Salesforce Connector, follow these steps:
 
 . In the *Mule Palette* view, select *Salesforce* and then select the desired operation.
 . Drag the operation onto the Studio canvas to the right of the source.
 
 [[configure-global-element]]
 == Configure a Global Element for the Connector
 
-When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Salesforce system.
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Salesforce system.
+
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
 
-To configure the global element for Salesforce Connector, follow these steps:
+To configure a global element for Salesforce Connector, follow these steps:
 
 . Select the name of the connector in the Studio canvas.
 . In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
-. On the *General* tab, configure the authentication connection type:
-* <<Basic Authentication>>
-* <<OAuth 2.0>>
-* <<OAuth JWT>>
-* <<OAuth SAML>>
-* <<OAuth Username Password>>
-. For each of the connection choices, you can set optional proxy values:
-+
-** *Host*
-+
-Host name of the proxy server
+. In the *General* tab, select one of these authentication methods for your connection:
 +
-** *Port*
-+
-Number of the port on which the proxy server runs
-+
-** *Username*
-+
-Username to log in to the server
+* <<basic,Basic authentication>>
+* <<oauth,OAuth 2.0 authentication>>
+* <<oauth-jwt,OAuth JWT authentication>>
+* <<oauth-saml,OAuth SAML authentication>>
+* <<oauth-username-password,OAuth Username Password authentication>>
+
 +
-** *Password*
+*Kim. Ideally the next two paragraphs would be in a section called Authentication, but that would make the current H3's for the authentication types into H4s. I would still like to do that unless you have a better idea. I also am having trouble getting these paragraphs to format correctly within the list.* 
```

Comment:
I think that you have to use === Authentication Methods, to be a peer to === Apex Settings.
I would not make === Mutual TLS a separate topic but rather include it in the intro text to Authentication Methods, the intro text stating what all four methods have in common: object store and mutual TLS.